_data/engine-cli-edge/docker_exec.yaml

```diff
@@ -41,53 +41,23 @@ options:
 - option: user
   shorthand: u
   description: 'Username or UID (format: <name|uid>[:<group|gid>])'
-examples: |-
-  ### Run `docker exec` on a running container
-
-  First, start a container.
-
-  ```bash
-  $ docker run --name ubuntu_bash --rm -i -t ubuntu bash
-  ```
-
-  This will create a container named `ubuntu_bash` and start a Bash session.
-
-  Next, execute a command on the container.
-
-  ```bash
-  $ docker exec -d ubuntu_bash touch /tmp/execWorks
-  ```
-
-  This will create a new file `/tmp/execWorks` inside the running container
-  `ubuntu_bash`, in the background.
-
-  Next, execute an interactive `bash` shell on the container.
-
-  ```bash
-  $ docker exec -it ubuntu_bash bash
-  ```
-
-  This will create a new Bash session in the container `ubuntu_bash`.
-
-  ### Try to run `docker exec` on a paused container
-
-  If the container is paused, then the `docker exec` command will fail with an error:
-
-  ```bash
-  $ docker pause test
-
-  test
-
-  $ docker ps
-
-  CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS                   PORTS               NAMES
-  1ae3b36715d2        ubuntu:latest       "bash"              17 seconds ago      Up 16 seconds (Paused)                       test
-
-  $ docker exec test ls
-
-  FATA[0000] Error response from daemon: Container test is paused, unpause the container before exec
-
-  $ echo $?
-  1
-  ```
+examples: "### Run `docker exec` on a running container\n\nFirst, start a container.\n\n```bash\n$
```

Comment:
Upstream looks like this;

```markdown
## Examples

### Run `docker exec` on a running container

First, start a container.
```

Looks like previously the yaml used 

```yaml
examples: |-
  ### Run `docker exec` on a running container
```
(and also had the literal output, not encoded newlines `\n`)


Wonder what changed, or if it was manually fixed previously